### PR TITLE
Fix test failures for nightly builds

### LINKED ISSUE #33: Fix test failures for nightly builds

### PR DESCRIPTION
**Pull request by TristanCacqueray**
This change fixes this cabal test  error:
  ghc: could not execute: hspec-discover

Fixes #33

### CODE REVIEW

**Comment by TristanCacqueray** [comment]
There are still `42` test failures with incorrect data, e.g.:

```
  test/OtherSpec.hs:70:7:
  40) Other, Faker Generate, App author
       expected: "Schmidt, Breitenberg and Lowe"
        but got: "Goldner, Will and Muller"
```

Should we fix the expected value? It seems like setDeterministic is not working...

**Comment by psibi** [comment]
@TristanCacqueray Thanks for the PR!

> Should we fix the expected value? It seems like setDeterministic is not working...

The reason the failure is happening is because of new version of random has changed the internals. So we should fix the failure so that the test case works for both the newer version of random library as well as older version of random library.

File: fakedata.cabal [hunk 1]
```diff
@@ -690,6 +690,7 @@ test-suite fakedata-test
   hs-source-dirs:
       test
   ghc-options: -threaded -rtsopts -with-rtsopts=-N
+  build-tool-depends: hspec-discover:hspec-discover
```
Comment (psibi) [inline fakedata.cabal:693]: Is it possible to specificy it via `package.yaml` ? We generate the cabal file via that.

**Review by psibi — CHANGES_REQUESTED** [review]

**Comment by psibi** [comment]
@TristanCacqueray If I understand, the underlying algorithm itself has changed between random versions. But if you can find a seed that works for both the case, then sure we can go ahead with that. :-)

**Comment by TristanCacqueray** [comment]
I looked for such seed with https://gist.github.com/TristanCacqueray/64795ba9444af8fe0314e44e59f14942 , but could not find one from 0 to 510_000. And even if such seed exists, it may not work for all the tests. So manually fixing the test seems like the best solution.

**Comment by psibi** [comment]
Seems it needs some adjustment in the CI scripts to make it pass (different stack.yaml for different resolvers). I'm going ahead and merging this - that can be fixed separately. Thanks @TristanCacqueray for the PR!

**Pull request merged**